tools/pydantic_codegen/automations/automations.toml

```diff
@@ -0,0 +1,39 @@
+# NOTE: Paths are relative to the root of the repo
+
+[tool.ariadne-codegen]
+schema_path = "core/api/graphql/schemas/schema-latest.graphql"
+queries_path = "tools/pydantic_codegen/automations/queries.graphql"
+
+files_to_include = ["tools/pydantic_codegen/base.py"]
+
+modules_to_drop = [
+    "async_base_client",
+    "base_client",
+    # We'll swap in a module with our own custom base class
+    "base_model",
+    "client",
+    "exceptions",
+]
```

Comment:
**note:** `modules_to_drop` is a custom added parameter, and isn't supported by `ariadne-codegen` out of the box